recipes/ruyaml/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "ruyaml" %}
+{% set version = "0.91.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ruyaml-{{ version }}.tar.gz
+  sha256: 6ce9de9f4d082d696d3bde264664d1bcdca8f5a9dff9d1a1f1a127969ab871ab
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - distro >=1.3.0
+    - python >=3.6
+    - setuptools >=39.0
+
+test:
+  imports:
+    - ruyaml
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/pycontribs/ruyaml
+  summary: ruyaml is a fork of ruamel.yaml
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - olegtarasov
```

Comment:
Ping @mbargull who are a contributor upstream and may want to be a maintainer here in the future.